islands/ItemsList.tsx

```diff
@@ -61,12 +63,17 @@ export default function ItemsList(props: { endpoint: string }) {
   }
 
   useEffect(() => {
+    if (!props.isSignedIn) {
+      loadMoreItems();
+      return;
+    }
+
     fetchVotedItems()
       .then((votedItems) =>
         votedItemsIdsSig.value = votedItems.map(({ id }) => id)
       )
       .finally(() => loadMoreItems());
-  }, []);
+  }, [props.isSignedIn]);
```

Comment:
Is this needed? I'm not sure it is.

Reply:
I removed this as I think it's unnecessary. If you think it's necessary, please let me know.

Reply:
shouldn't be needed, thanks for removing!